Description: Use system tinyxml library instead of building a local copy
Author: Felix Geyer <fgeyer@debian.org>

---
 src/CMakeLists.txt               |    1 -
 src/FileRead.cpp                 |    2 +-
 src/ReplayRecorder.cpp           |    2 +-
 src/TextManager.cpp              |    2 +-
 src/UserConfig.cpp               |    2 +-
 src/state/NetworkSearchState.cpp |    2 +-
 6 files changed, 5 insertions(+), 6 deletions(-)

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,4 +1,3 @@
-add_subdirectory(tinyxml)
 ADD_DEFINITIONS(-std=c++11)
 add_subdirectory(raknet)
 add_subdirectory(blobnet)
--- a/src/FileRead.cpp
+++ b/src/FileRead.cpp
@@ -30,7 +30,7 @@ Foundation, Inc., 59 Temple Place, Suite
 #include <boost/scoped_array.hpp>
 #include <boost/algorithm/string.hpp>
 
-#include "tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 extern "C"
 {
--- a/src/ReplayRecorder.cpp
+++ b/src/ReplayRecorder.cpp
@@ -29,7 +29,7 @@ Foundation, Inc., 59 Temple Place, Suite
 
 #include <boost/crc.hpp>
 
-#include "tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 #include "raknet/BitStream.h"
 
--- a/src/TextManager.cpp
+++ b/src/TextManager.cpp
@@ -26,7 +26,7 @@ Foundation, Inc., 59 Temple Place, Suite
 #include <algorithm>
 #include <set>
 
-#include "tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 #include "Global.h"
 #include "FileRead.h"
--- a/src/UserConfig.cpp
+++ b/src/UserConfig.cpp
@@ -27,7 +27,7 @@ Foundation, Inc., 59 Temple Place, Suite
 
 #include <boost/make_shared.hpp>
 
-#include "tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 #include "Global.h"
 #include "FileRead.h"
--- a/src/state/NetworkSearchState.cpp
+++ b/src/state/NetworkSearchState.cpp
@@ -36,7 +36,7 @@ Foundation, Inc., 59 Temple Place, Suite
 #include "blobnet/layer/Http.hpp"
 #include "blobnet/exception/HttpException.hpp"
 
-#include "tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 #include "NetworkState.h"
 #include "LobbyState.h"
